oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxHardwareAbstractionLayer.java

```diff
@@ -18,16 +18,7 @@
  */
 package oshi.hardware.platform.linux;
 
-import oshi.hardware.CentralProcessor;
-import oshi.hardware.ComputerSystem;
-import oshi.hardware.Display;
-import oshi.hardware.GlobalMemory;
-import oshi.hardware.HWDiskStore;
-import oshi.hardware.NetworkIF;
-import oshi.hardware.PowerSource;
-import oshi.hardware.Sensors;
-import oshi.hardware.SoundCard;
-import oshi.hardware.UsbDevice;
+import oshi.hardware.*;
```

Comment:
FYI, my formatter will expand this again.  While this compiles if you can set up your formatter to list each import individually it'll save later spurious changes on commits.